pkgs/stdenv/generic/default.nix

```diff
@@ -127,6 +127,11 @@ let
         "`stdenv.isArm` is deprecated after 18.03. Please use `stdenv.isAarch32` instead"
         hostPlatform.isAarch32;
 
+      system = builtins.trace
```

Comment:
I think we want to use lib.warn here where possible.

Reply:
Thanks! Fixed in https://github.com/NixOS/nixpkgs/pull/46059